# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1227166485 -3600
#      Thu Nov 20 08:34:45 2008 +0100
# Branch release-3-0-x
# Node ID a0b28e51e3d2c6161f6f870cb4abaf94100337eb
# Parent  7f07a6cc41aed907fc190a977cf19aa890dbc235
spline.m: delete debugging statements

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2008-11-10  John W. Eaton  <jwe@octave.org>
+
+	* polynomial/spline.m: Delete debugging statements.  From
+	Sebastian Schöps <sebastian@schoeps.org>.
+
 2008-11-13  John W. Eaton  <jwe@octave.org>
 
 	* statistics/distributions/chi2rnd.m: Fix another missing semicolon.
 	From sven.mattisson@insatnet.nu.
 
 2008-10-08  John W. Eaton  <jwe@octave.org>
 
 	* miscellaneous/fileparts.m: Handle "/file" properly.
diff --git a/scripts/polynomial/spline.m b/scripts/polynomial/spline.m
--- a/scripts/polynomial/spline.m
+++ b/scripts/polynomial/spline.m
@@ -120,20 +120,16 @@ function ret = spline (x, y, xi)
       c(2:n-1,:) = 1/dg(1);
     else
       dg = 2 * (h(1:n-2) .+ h(2:n-1));
       dg(1) = dg(1) - 0.5 * h(1);
       dg(n-2) = dg(n-2) - 0.5 * h(n-1);
 
       e = h(2:n-2);
 
-      size(a)
-      size(h)
-      n
-
       g = 3 * diff (a(2:n,:)) ./ h(2:n-1,idx) ...
           - 3 * diff (a(1:n-1,:)) ./ h(1:n-2,idx);
       g(1,:) = 3 * (a(3,:) - a(2,:)) / h(2) ...
           - 3 / 2 * (3 * (a(2,:) - a(1,:)) / h(1) - dfs);
       g(n-2,:) = 3 / 2 * (3 * (a(n,:) - a(n-1,:)) / h(n-1) - dfe) ...
           - 3 * (a(n-1,:) - a(n-2,:)) / h(n-2);
 
       c(2:n-1,:) = spdiags ([[e(:); 0], dg, [0; e(:)]],
